### PR TITLE
Rename to voxpupuli

### PR DESCRIPTION
**Pull request by petems**

### CODE REVIEW

**Comment by bastelfreak** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 The rundeck puppet module for installing and managing rundeck (http://rundeck.org/)
 
-[![Build Status](https://travis-ci.org/puppet-community/puppet-rundeck.svg?branch=master)](https://travis-ci.org/puppet-community/puppet-rundeck)
+[![Build Status](https://travis-ci.org/voxpupuli/puppet-rundeck.svg?branch=master)](https://travis-ci.org/voxpupuliy/puppet-rundeck)
```
Comment (jyaworski) [inline README.md]: Spelling error. :) Bad @bastelfreak.

Comment (bastelfreak) [inline README.md]: fix in https://github.com/voxpupuli/puppet-rundeck/pull/170

Comment (petems) [inline README.md]: :sob:

**Pull request merged**